ds_judgements_public_ui/templates/includes/judgment_catalogue_card.html

```diff
@@ -17,15 +17,12 @@ <h2>Document summary information</h2>
       {% if document.body.respondent %}
         <tr><th>Respondent</th><td>{{ document.body.respondent }}</td></tr>
       {% endif %}
-      {% if document.body.category %}
-        <tr><th>Category</th><td>{{ document.body.category }}</td></tr>
-      {% endif %}
-      {% if document.body.subcategory %}
-        <tr><th>Subcategory</th><td>{{ document.body.subcategory }}</td></tr>
-      {% endif %}
-      {% if document.body.secondary_subcategory %}
-        <tr><th>Secondary subcategory</th><td>{{ document.body.secondary_subcategory }}</td></tr>
-      {% endif %}
+      {% for category in document.body.categories %}
+        <tr><th>Category</th><td>{{ category.name }}</td></tr>
+        {% for subcategory in category.subcategories %}
+          <tr><th>Subcategory</th><td>{{ subcategory.name }}</td></tr>
```

Comment:
There's a part of me wondering whether to give this a fraction of an indent. Not essential.